app/ui/src/main/java/com/fsck/k9/activity/K9Activity.java

```diff
@@ -29,4 +34,24 @@ public boolean dispatchTouchEvent(MotionEvent event) {
     public void setupGestureDetector(OnSwipeGestureListener listener) {
         mBase.setupGestureDetector(listener);
     }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode,
+                                           String permissions[], int[] grantResults) {
+        switch (requestCode) {
+            case PERMISSIONS_REQUEST_READ_CONTACTS:
+            case PERMISSIONS_REQUEST_WRITE_CONTACTS: {
+                // If request is cancelled, the result arrays are empty.
+                if (grantResults.length > 0
+                        && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
+
+                    Toast.makeText(this, R.string.contact_permission_thanks,
```

Comment:
seems unnecessary

Reply:
The thanks message? I just left that in from the orignal pr, I will remove it.